src/main/java/it/gov/pagopa/pu/classification/mapper/TreasuredClassificationMapper.java

```diff
@@ -0,0 +1,128 @@
+package it.gov.pagopa.pu.classification.mapper;
+
+import it.gov.pagopa.pu.classification.dto.generated.PagedTreasuredClassification;
+import it.gov.pagopa.pu.classification.dto.generated.TreasuredClassificationDTO;
+import it.gov.pagopa.pu.classification.model.view.TreasuredClassification;
+import java.util.Collections;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TreasuredClassificationMapper {
+
+  public TreasuredClassificationDTO map2DTO(TreasuredClassification entity) {
```

Comment:
Could you typeMap `TreasuredClassification` instead? this is an @Entity and it's allowed to be typeMapped